packages/spec/jest.setup.js

```diff
@@ -1 +1,20 @@
 jest.setTimeout(5 * 60 * 1000);
+
```

Comment:
I pulled the check of the console output out of `createPage`, because it needs to be configurable.

Another approach might be, to pass in a path to module, that exports an `isIntolerableWarning`-function, via the [`testEnvironmentOptions`](https://jestjs.io/docs/en/configuration#testenvironmentoptions-object), import that module inside the custom environment & thereby keep the actual checking encapsulated.

WDYT?

Reply:
I think it's ok for now until we have more usages and can find a better solution.